app/helpers/banner_helper.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require "yaml"
+
+module BannerHelper
+  # rubocop:disable Rails/ContentTag
+  def banner_content
+    @yaml_data = YAML.load_file("config/banner.yml")
+    return false if @yaml_data[Rails.env].nil?
+    @banner_title = "<h1>#{@yaml_data[Rails.env]['title']}</h1>"
```

Comment:
I would love it if the `<h1>` wen into the partial
```suggestion
    @banner_title = @yaml_data[Rails.env]['title']
```